packages/amazonq/.changes/next-release/Feature-93485fee-05c0-4dc9-abde-9072590cbd45.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Feature",
+	"description": "Enable default `@workspace` context of Amazon Q chat for certain users"
```

Comment:
if it's not for public users to access this feature, i think we probably dont want to mention it?

Reply:
but it is still customer facing